padle/padle.py

```diff
@@ -0,0 +1,801 @@
+            if not m.on_na or m.name_en is None:
+                result.append(f"{id} is not a monster on the NA server.")
+                continue
+            if int(id) in monsters_list:
+                result.append(
+                    f"{MonsterHeader.menu_title(m, use_emoji=True).to_markdown()} is already a possible PADle.")
+                continue
+            valids.append(int(id))
+            result.append(f"{MonsterHeader.menu_title(m, use_emoji=True).to_markdown()} was added.")
+        async with self.config.monsters_list() as m_list:
+            m_list.extend(valids)
+        for page in pagify("\n".join(result)):
+            await ctx.send(page)
+
+    @padleadmin.command()
+    async def remove(self, ctx, *, list):
+        """Removes CSV of monster IDs to the list of possible PADles"""
+        clean_list = re.sub(" ", "", list).split(",")
+        dbcog = await self.get_dbcog()
+        result = []
+        valids = []
+        monsters_list = await self.config.monsters_list()
+        for id in clean_list:
+            if not id.isnumeric():
+                result.append(f"{id} is not a valid monster ID.")
+                continue
+            m = dbcog.get_monster(int(id))
+            if m is None:
+                result.append(f"{id} is not a valid monster ID.")
+                continue
+            if not m.on_na or m.name_en is None:
+                result.append(f"{id} is not a monster on the NA server.")
+                continue
+            if int(id) not in monsters_list:
+                result.append(f"{MonsterHeader.menu_title(m, use_emoji=True).to_markdown()} is not a possible PADle.")
+                continue
+            valids.append(int(id))
+            result.append(f"{MonsterHeader.menu_title(m, use_emoji=True).to_markdown()} was removed.")
+        await self.config.monsters_list.set([m for m in monsters_list if m not in valids])
+        for page in pagify("\n".join(result)):
+            await ctx.send(page)
+
+    @padleadmin.command()
+    async def set(self, ctx):
+        """Set possible PADles to attached CSV/text file"""
+        if ctx.message.attachments:
+            try:
+                data = [int(x) for x in
+                        re.sub("[^0-9,]", "", (await ctx.message.attachments[0].read()).decode("utf-8")).split(",")]
+                dbcog = await self.get_dbcog()
+                for id in data:
+                    m = dbcog.get_monster(int(id))
+                    if m is None or not m.on_na or m.name_en is None:
+                        return await send_cancellation_message(ctx, f"{id} was invalid. Please remove "
+                                                                    "this ID and try again.")
+                await self.config.monsters_list.set(data)
+                await ctx.tick()
+            except Exception as e:
+                await send_cancellation_message(ctx, "Something went wrong.")
+                await ctx.send(box(str(e)))
```

Comment:
You should log this rather than sending it to the user.  `logger.exception()` or whatever